lib/config.js

```diff
@@ -249,6 +247,8 @@ Config.prototype.update = function (options) {
     this.officialBuild = this.buildConfig === 'Release'
   }
 
+  this.sccache = (!this.officialBuild || process.platform !== 'win32') && getNPMConfig(['sccache'])
```

Comment:
I don't think we need the extra checks here, it's covered https://github.com/brave/brave-browser/pull/644/files#diff-f7bf2b665273dfa66ffa4ad7c0a52bb9R98
and the officialBuild check isn't necessary at all anymore
also can you remove https://github.com/brave/brave-browser/pull/644/files#diff-f7bf2b665273dfa66ffa4ad7c0a52bb9R109 while you're here?